packagingenv/trusty/Dockerfile

```diff
@@ -38,6 +38,9 @@ RUN echo -e "#!/bin/sh\nexit 101\n" > /usr/sbin/policy-rc.d && \
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
     apt-get -y install build-essential python-dev python
 
+# install netbase package (includes /etc/protocols and other files we rely on)
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
```

Comment:
Please remove another duplicated `apt-get update`. We already have a lot of them.

Reply:
Yeah, wasn't sure about it - I saw it on every line so I just followed the existing approach.